superset-frontend/src/dashboard/components/Header.jsx

```diff
@@ -482,7 +482,7 @@ class Header extends React.PureComponent {
             <>
               <span
                 role="button"
-                title="Edit dashboard"
+                title={t('Edit Dashboard')}
```

Comment:
We use sentence case for most UI elements: https://github.com/apache/superset/blob/master/CONTRIBUTING.md#capitalization-guidelines

Reply:
@ktmud OK~
I've changed `Edit Dashboard` to `Edit dashboard`